Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -37,6 +40,10 @@ import BraintreeCore
     /// We require a static reference of the client to call `handleReturnURL` and return to the app.
     static var venmoClient: BTVenmoClient? = nil
 
+    /// Used for linking events from the client to server side request
+    /// In the PayPal flow this will be either an EC token or a Billing Agreement token
+    private var payPalContextID: String? = nil
```

Comment:
Maybe more clear to name this `paymentContextID` instead, and store it directly from line 176

Reply:
I thought it may make it more clear why we are storing it by calling it `payPalContextID`, but am fine with either name. We aren't able to store it until after it's initialized by the guard, though. It's a bit goofy for sure but we'd get a compiler error. Thoughts?